src/ProjectLayout/MemberForm.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react"
+
+export default function MemberForm({members, setMembers}) {
+    const [member, setMember] = useState('');
+    
+  function handleSubmit(e){
+    e.preventDefault();
+    setMembers([
+      {
+        id: members.length + 1,
+        name: member,
+      },
+        ...members
+      ]);
+      e.target.reset();
+    }
+    
+  return (
+    <div>
+      <form onSubmit={e => handleSubmit(e)}>
+        <input 
+        type='text' 
+        name='memberName' 
+        id='memberName' 
+        placeholder='Add team Member(s)'
+        onChange={e => setMember(e.target.value)}
+        />
+        <button type='submit' id='add-member-btn'>Add</button>
+      </form>
+      <div className='chips-container'>
+        { members.map(myMember => (
```

Comment:
@Tom-Sarfo you should check for the availability of a variable before mapping on it. You can use optional chaining in this scenario.

Reply:
Okay sure.